config.json

```diff
@@ -1305,6 +1305,15 @@
         "nugetId": "Xamarin.Android.Volley.CroNet",
         "dependencyOnly": false
       },
+      {
```

Comment:
This entry is duplicated, version `2.0.0.2` is also in this file.

Reply:
seems like manual merge error

Reply:
fixed